packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageList.tests.js

```diff
@@ -947,14 +968,14 @@ QUnit.module('MessageList', moduleConfig, () => {
             setTimeout(() => {
                 const scrollTop = this.getScrollView().scrollTop();
 
-                assert.roughEqual(scrollTop, this.getScrollOffsetMax(), 1, 'scroll position should be at the bottom after initialization');
+                assert.roughEqual(scrollTop, this.getScrollOffsetMax(), 1.2, 'scroll position should be at the bottom after initialization');
```

Comment:
Just to clarify: Do these tests fail on your local machine?

Reply:
yeap, I believe that's because of OS zoom

Reply:
Yep, most of the time, it may be related to zoom settings. Since then even 1.2 proved to be not enough. I will try to run the tests with default system and browser zoom, just to be sure that's the problem.

Reply:
Ran these tests with 100% OS zoom and they all pass with the original settings. Should I revert the 1.2?

Reply:
I don’t have any objections to increasing the epsilon here. 
JFYI: In the future, it’s better to avoid changes that don’t relate to the PR. We can refactor or enhance some parts in separate tasks after its planning